tooling/minimize/src/constant.rs

```diff
@@ -31,6 +32,38 @@ fn translate_const_val<'cx, 'tcx>(
         Type::Tuple { fields, .. } if fields.is_empty() => {
             return ValueExpr::Tuple(List::new(), ty);
         }
+        Type::Enum { size, align, .. } => {
+            assert!(size.bytes() <= 16);
+            let Some(bits) = val.try_to_bits_for_ty(fcx.cx.tcx, rs::ParamEnv::reveal_all(), rs_ty) else {
+                panic!("Can only create constant enum from bits. Got {:?}", val);
+            };
+            // According to the documentation the first size bytes are data.
+            // This suggests BigEndian but is unclear.
```

Comment:
This is in target endianess, i.e., `fcx.cx.tcx.data_layout().endian`. But we shouldn't have to care about that, ideally...